src/cascadia/TerminalSettingsEditor/EditColorScheme.xaml

```diff
@@ -450,6 +450,39 @@
                     </Button>
                 </StackPanel>
             </local:SettingContainer>
+
+            <local:SettingContainer x:Name="SetAsDefaultContainer"
+                                    x:Uid="ColorScheme_SetAsDefault">
+                <Button x:Uid="ColorScheme_SetAsDefaultButton"
+                        Click="{x:Bind ViewModel.SetAsDefault_Click}"
+                        Style="{StaticResource BrowseButtonStyle}" />
+            </local:SettingContainer>

```

Comment:
Ugh, this is really annoying so I'm sorry. Because of #14398, we need the `AutomationProperties.Name` to match the displayed text for this button. Since `SettingContainer` automatically applies the `Header` resource as the name, the name no longer matches the displayed text.

Could you please...
- add an `x:Name` to this button
- In `EditColorScheme.cpp::EditColorScheme()`, explicitly set the AP.Name to the displayed text

I _think_ that should do the trick, but if it doesn't, we'll have to fix it either as a part of this PR or in a follow-up before release.

Reply:
Done!